client-common/src/main/java/org/apache/cassandra/sidecar/common/ApiEndpointsV1.java

```diff
@@ -121,6 +121,12 @@ public final class ApiEndpointsV1
     public static final String LIST_CDC_SEGMENTS_ROUTE = API_V1 + CDC_PATH + "/segments";
     public static final String STREAM_CDC_SEGMENTS_ROUTE = LIST_CDC_SEGMENTS_ROUTE + "/" + SEGMENT_PATH_PARAM;
 
+    public static final String SERVICES_PATH = "/services";
+    public static final String SERVICE_PARAM = ":service";
+    public static final String CONFIG = "/config";
+    public static final String SERVICE_CONFIG_ROUTE = API_V1 + SERVICES_PATH + SERVICE_PARAM + CONFIG;
+    public static final String GET_SERVICES_CONFIG_ROUTE = API_V1 + SERVICES_PATH;
```

Comment:
I think the URI should be the same for the service, with different verbs to specify the action:

`/api/v1/config/services/:service`

- GET : retrieve configuration for :service
- DELETE: remove configuration for :service
- PUT: update configuration for :service

Reply:
The get verb gets config for all the services, I think it is valuable. If we want to be able to get configs of a given service we can add GET verb on `/api/v1/config/services/:service`.